Formula/ghc.rb

```diff
@@ -25,6 +26,8 @@ class Ghc < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "66de55e420c039535422aed4c7803285faedf5292dd2385f6b7d7bcc5d899d94"
   end
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
   depends_on "python@3.10" => :build
```

Comment:
Might switch this to Python 3.11 (though Python 3.10 is still installed at build due to `sphinx-doc`):
```suggestion
  depends_on "python@3.11" => :build
```